resources/js/register_helper.js

```diff
@@ -1,7 +1,4 @@
 $(document).ready(function() {
```

Comment:
BTW, I think this file should be renamed.

Reply:
Actually now it only contains the datepicker initializer (8 lines). we can have them in the blade, I think.

Reply:
Sure.